books/RayTracingInOneWeekend.html

```diff
@@ -483,23 +483,15 @@
 <div class='together'>
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
 calculating whether a ray hits a sphere is pretty straightforward. Recall that the equation for a
-sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 = R^2$. The way you can read that
-equation is “for any $(x, y, z)$, if $x^2 + y^2 + z^2 = R^2$ then $(x,y,z)$ is on the sphere, and
-otherwise it is not”. It gets uglier if the sphere center is at
-$(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+sphere centered at the origin is $x^2 + y^2 + z^2 = R^2$.
```

Comment:
What's the rationale for dropping the explanation of `R`?

Reply:
i'll give this a shot maybe later today.

Reply:
In high school, one of my English teachers discussed the topic of, I can't remember the term that was used, but the idea was about efficiency.
"An example of [inefficiency] is 'dog puppies.'"

This idea stuck with me.  If I've proofread something I typed enough times, imo it's a pretty easy read.

Efficiency isn't all there is in life, of course.  I had a "logic"-interested friend in college who asked our philosophy professor "Isn't the goal of language to be efficient?"

"The primary goal of a language should be to be beautiful."

Efficiency tends to have priority in my life.

My thoughts on the first paragraph:

"Let's add a single object to our ray tracer.  People often use spheres in ray tracers because .."
Nice.

"Recall that the equation for a sphere centered at the origin of radius R is x2 + y2 + z2 = R2."
OK he just said R twice.  Let's axe "of Radius R"

"The way you can read that equation is 'for any (x,y,z), if x2 + y2 + z2 = R2'.."
Another thing said twice.

"..then (x,y,z)"
hmm

"..is on the sphere, and otherwise it is not."
binary logic.  the other half is implied.  axe that and keep reading

"It gets uglier if the sphere center is at (cx,cy,cz)."
level of understanding:  instant

After reading the second paragraph, I saw that (cx,cy,xz) became represented as 'c' so I choose to 'c' (cx,cy,cz) in first paragraph.

At this point I look again at "The way you can read that equation .. otherwise it is not."
Look like dragon's breath.  x,y & z are already implied from the rather simple and beautiful looking equations that follow.  "Axe the whole sentence."

Reply:
> "Recall that the equation for a sphere centered at the origin of radius R is x2 + y2 + z2 = R2."
OK he just said R twice. Let's axe "of Radius R"

If I give you the equation n₁sinθ₁ = n₂sinθ₂, it's worthless unless I then tell you what n₁, θ₁, n₂ and θ₂ all _mean_. It's not repetition to say “with incident angle θ₁”.

I agree with the comment about not eliding the “otherwise false” phrases.

Reply:
if one took high school algebra there's a ridiculously slim chance of the need to explain what R represents

Reply:
yeah i'm a hot-head.  sorry, i'm trying.  with what you bring up, and 'non double-speak' one could say x2 + y2 + z2 = Radius2